test/test_ops.py

```diff
@@ -84,6 +88,10 @@
 
 assert torch.get_default_dtype() == torch.float32
 
+
+enable_backend_test(op_db)
+enable_backend_test(python_ref_db)
+
```

Comment:
I think the code should check if current test is dedicated to XPU. Otherwise, we do not need to enable backend test.
```suggestion

if TEST_XPU:
    apply_op_db_for(op_db, device="xpu")
    apply_python_ref_op_db_for(python_ref_db, device="xpu")

```

Reply:
I did the 'if TEST_XPU' check in enable_backend_test() function. Is it ok?